tests/src/runner/podman-desktop-runner.ts

```diff
@@ -223,7 +245,7 @@ export class PodmanDesktopRunner {
     this._options = value;
   }
 
-  public setVideoName(name: string) {
+  public setVideoAndTraceName(name: string) {
```

Comment:
Please also update the class member property `videoName`.

Reply:
done